src/renders/refreshScoresTab.js

```diff
@@ -0,0 +1,27 @@
+import renderScore from './scoreRow.js';
+
+export default async () => {
```

Comment:
- [ ] The submit button should hit the POST API endpoint and post user entries, and when the Refresh button is clicked, it should hit the GET API endpoint causing the scores list to change, kindly consider separating the action from the submit button so it just send the data not refreshing the score list